pallets/frequency-tx-payment/src/benchmarking.rs

```diff
@@ -0,0 +1,20 @@
+use super::*;
+
+use frame_benchmarking::{benchmarks, whitelisted_caller};
+use frame_system::{Call as SystemCall, RawOrigin};
+
+benchmarks! {
+	pay_with_capacity {
+		let sender: T::AccountId = whitelisted_caller();
+		let call: Box<<T as Config>::RuntimeCall> = Box::new(SystemCall::remark { remark: vec![] }.into());
+	}: _ (RawOrigin::Signed(sender), call)
+	verify {
```

Comment:
We can remove this for now. After discussing with Shannon, we decided to add an event when withdrawing the fee.